Makefile

```diff
@@ -77,6 +78,10 @@ gobuild:
 docker:
 	@[ ! -z "$(PIPELINE)" ] || docker build --build-arg=GOPROXY=${GOPROXY} --file build/Dockerfile -t "$(BUILD_IMAGE)" .
 
+.PHONY: dockerx
+dockerx:
+	@[ ! -z "$(PIPELINE)" ] || docker buildx build --progress=plain --build-arg=GOPROXY=${GOPROXY} --platform=linux/arm64,linux/amd64 --file build/Dockerfile -t "$(BUILD_IMAGE)" .
```

Comment:
Shouldn't `./.ci/publish-images.sh` be changed to use this new target?

Reply:
Not for this PR, but how hard would it be to add s390? Is it just a matter of adding a new platform, or would it typically imply other changes as well?

Reply:
golang support s390x.
so just added platform linux/s390x.
without this pr

old way will build in qemu, which will be very slow. this pr is for fixing this

Reply:
it should, i missing the changes, update it later